npm/private/npm_translate_lock_state.bzl

```diff
@@ -104,6 +106,10 @@ def _init_common_labels(rctx, label_store):
     # pnpm-workspace.yaml file
     label_store.add_sibling("lock", "pnpm_workspace", PNPM_WORKSPACE_FILENAME)
 
+    # verify patches list
+    if attr.verify_patches:
```

Comment:
nit: could move this one to within `_init_patch_labels` since it is related to patches